ground/gcs/src/app/main.cpp

```diff
@@ -263,8 +263,15 @@ int main(int argc, char **argv)
         // fix Mac OS X 10.9 (mavericks) font issue
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
+
+        QSettings tmpSettings;
+
+        bool value = true;
+
```

Comment:
I'd like to see documentation here. This code is opaque and hard to maintain code if there's no description about what it does.